content/blog/series/gsod-2020-vocabulary-usage-guide/contents.lr

```diff
@@ -0,0 +1 @@
+name: GSOD-2020: Vocabulary Usage Guide
```

Comment:
I'm afraid the second colon will be misinterpreted unless we quote it. I could be wrong.

```suggestion
name: "GSoD-2020: Vocabulary Usage Guide"
```

Reply:
Oh, okay alright!